js/views/pageNavVw.js

```diff
@@ -66,6 +67,12 @@ module.exports = baseVw.extend({
       this.model.fetch();
     });
 
+    ipcRenderer.on('goto', function(event, arg) {
+      self.addressBarProcess(arg);
+    });
+
+    ipcRenderer.send('checkURI', '');
+
```

Comment:
Ok I'm not sure how to address this. Is @rmisio or @jjeffryes going to handle this instead?